WeakAuras/Prototypes.lua

```diff
@@ -9498,7 +9500,7 @@ Private.event_prototypes = {
         type = "number",
         init = "select(2, UnitResistance('player', 3))",
```

Comment:
does UnitResistance work on cata?

Reply:
On current build it does

Reply:
It works, but it may be one of the things blizzard forgot to remove and will be drop later